core/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust:1.70.0 as core
```

Comment:
Is that what you are running locally ? I think I'm using 1.73

Reply:
Yes this is my local and prod was 1.68